docs/sources/next/using-k6-browser/migrating-to-k6-v0-52.md

```diff
@@ -0,0 +1,309 @@
+---
+title: 'Migrating browser scripts to k6 v0.52'
+description: 'A migration guide to ease the process of transitioning to the new non-experimental k6 browser module version bundled with k6 v0.52'
+weight: 04
+---
+
+# Migrating browser scripts to k6 v0.52
+
+This guide outlines the key changes you will need to make when moving your existing k6 browser test scripts to the _latest_ non-experimental [k6 browser module](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser) version bundled with [k6 0.52](https://github.com/grafana/k6/releases/tag/v0.52.0).
```

Comment:
We also use "non-experimental" a few more times in other places in the migration doc. Maybe we should say it once at the beginning of the documentation, something like "the browser module is no longer an experimental module starting with k6 v0.52.0". Then, we might just use the k6 0.52 beyond without saying non-experimental. Also, the import path no longer contains the word "non-experimental." It might be confusing.

Reply:
I see what you are getting at. I've updated it in 0e3acc5c2efb5ee9876ecff41ee82b94fbb49015

Reply:
Looks great. Thanks 🙇